src/foremast/consts.py

```diff
@@ -139,6 +141,53 @@ def find_config():
 
     return configurations
 
+def _remove_empty_entries(entries):
+    """Remove emtpy entries in a list"""
+    for count, entry in enumerate(entries):
+        if entry in ['', None]:
+            entries.pop(count)
+    return entries
+
+
+def _convert_string_to_native(value):
+    """Convert a string to its native python type"""
+    result = None
+
+    try:
+        result = ast.literal_eval(value)
```

Comment:
I don't know if we need `ast` here. If we're possibly expecting JSON, maybe we should use `json`? A test case for this sole function would help clarify what the incoming data is.